nemo_skills/inference/merge_chunks.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
why not do it in python? Much easier to check things, understand what's happening and modify logic. You can still call system cat from python

Reply:
converted in [c567f63](https://github.com/NVIDIA/NeMo-Skills/pull/350/commits/c567f63e714787f5fd08c61734f7d70aeff0ca04)